assets/app/data/crowbar/services/crowbar-backup.factory.js

```diff
@@ -23,9 +23,9 @@
 
             var requestOptions = {
                 method: 'POST',
-                url: '/utils/backups',
```

Comment:
Actually I withdrew this comment. 
Naming services by URL component doesn't seem to work very well for us.
Changes in API structure lead to unnecessary code changes on UI side which can introduce additional errors.
`crowbarBackupFactory` is a great example: it uses `/utils/backups` and `/api/upgrade/adminbackup` which with our current/old approach would cause these functions to be located in two services. IMO it's better to have them grouped by the functional area they cover (i.e. backups in this case).